misc/config_tools/schema/types.xsd

```diff
@@ -203,6 +203,14 @@ RDT, setting this option to ``y`` is ignored.</xs:documentation>
         <xs:documentation>Specify whether to enable Code and Data Prioritization (CDP).
 CDP is an extension of CAT. Set to 'y' to enable the feature or 'n' to disable it.
 The 'y' will be ignored when hardware does not support CDP. Default
+value ``n``.</xs:documentation>
+      </xs:annotation>
+    </xs:element>
+    <xs:element name="VCAT_ENABLED" type="Boolean" default="n">
+      <xs:annotation>
+        <xs:documentation>Specify whether to enable CAT virtualization (vCAT).
+Set to 'y' to enable the feature or 'n' to disable it.
+The 'y' will be ignored when hardware does not support CAT. Default
```

Comment:
Same comment here.  You should delete mentioning what the default value is in this annotation.  The doc generation process will add the default value if it's specified in the element (see line 209), so if you add it in the annotation it will show up twice.

Reply:
Sure, will delete the mentioning of the default value

Reply:
Fixed and updated this PR